google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -78,7 +78,7 @@ appengine.flex.configtype.custom.label=Custom
 appengine.flex.user.specified.artifact.label=Select a deployment archive:
 appengine.flex.dockerfile.location.label=Dockerfile path\:
 appengine.flex.version.placeholder.text=auto generated
-appengine.flex.facet.config.title=App Engine flexible configuration
+appengine.flex.facet.config.title=App Engine Flexible Configuration
```

Comment:
This appears as a panel title, so I think its ok to have "Flexible" capitalized. Feel free to disagree.